lale/lib/sklearn/logistic_regression.py

```diff
@@ -45,7 +45,10 @@ def predict_proba(self, X):
     'X': {
       'description': 'Features; the outer array is over samples.',
       'type': 'array',
-      'items': {'type': 'array', 'items': {'type': 'number'}}},
+      'items': {'anyOf':
+          [{'type': 'array', 'items': {'type': 'number'}},
+           {'type': 'number'}
+          ]}},
```

Comment:
Hmm ... according to the scikit-learn documentation, the X argument to the LR.fit() method is always two-dimensional:

https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression.fit